rpc-broker/src/rpc-broker.h

```diff
@@ -0,0 +1,135 @@
+/*
+ Copyright (c) 2018 AIS, Inc.
+
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program; if not, write to the Free Software
+ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+*/
+
+#define _GNU_SOURCE
+#include <ctype.h>
+#include <dbus/dbus.h>
+#include <fcntl.h>
+#include <netinet/in.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <syslog.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <uv.h>
+
+#ifdef HAVE_XENSTORE
+#include <xenstore.h>
+#endif
+
+#include "rpc-dbus.h"
+#include "rpc-json.h"
+#include "policy.h"
+#include "signature.h"
+#include "websockets.h"
+
+
+#define DBUS_BROKER_ERROR(call)                                       \
+    do {                                                              \
+        openlog(NULL, LOG_PERROR, 0);                                 \
```

Comment:
Shouldn't `openlog()` be called only once?

Reply:
If anything triggers `DBUS_BROKER_ERROR` the program calls `exit` so thats pretty much it.